api_app/analyzers_manager/observable_analyzers/dns0/dns0_base.py

```diff
@@ -0,0 +1,133 @@
+import re
+from abc import ABCMeta
+from logging import getLogger
+
+import dateparser
+
+from api_app.analyzers_manager.classes import BaseAnalyzerMixin
+from api_app.analyzers_manager.exceptions import (
+    AnalyzerConfigurationException,
+    AnalyzerRunException,
+)
+
+_supported_sort_types = [
+    "first_seen",
+    "last_seen",
+]
+
+_min_limit_value = 0
+_max_limit_value = 50000
+
+_min_offset_value = 0
+
+logger = getLogger(__name__)
+
+
+class DNS0Mixin(BaseAnalyzerMixin, metaclass=ABCMeta):
+    base_url: str = "https://api.dns0.eu/"
+
+    _api_key: str
+    from_date: str
+    to_date: str
+    not_before: str
+    sort: str
+    format: str
+    limit: int
```

Comment:
I would explicitly put a default here of `100` even if that is a default of the API too. This is to be more explicit about how many data we are going to retrieve to the users.